requirements/edx/custom.txt

```diff
@@ -5,16 +5,16 @@
 -e git+https://github.com/edx-solutions/xblock-image-explorer.git@v1.0.1#egg=xblock-image-explorer==1.0.1
 -e git+https://github.com/edx-solutions/xblock-drag-and-drop.git@92ee2055a16899090a073e1df81e35d5293ad767#egg=xblock-drag-and-drop
 # FIXME: temp hash, until a proper solution is findout (ref ticket: https://edx-wiki.atlassian.net/browse/MCKIN-8249)
--e git+https://github.com/edx-solutions/xblock-drag-and-drop-v2.git@a4dee0689863533948735b4b389493c42b5110f0#egg=xblock-drag-and-drop-v2==2.2.0
+-e git+https://github.com/edx-solutions/xblock-drag-and-drop-v2.git@7c95f906af2f16eb23352b155c77a16466602631#egg=xblock-drag-and-drop-v2==2.2.1
 # This is required for A2E courses that were created with the temporary (xblock-drag-and-drop-v2-new) DnDv2 branch to continue to work.
 # FIXME: bump version to 2.1.6 when https://github.com/edx-solutions/xblock-drag-and-drop-v2/pull/154 merged and tagged
 -e git+https://github.com/open-craft/xblock-drag-and-drop-v2.git@82c9dc5e16d10793e8b79e60661e1a78893fce25#egg=xblock-drag-and-drop-v2-new
 -e git+https://github.com/edx-solutions/xblock-ooyala.git@v2.0.18#egg=xblock-ooyala==2.0.18
 git+https://github.com/edx-solutions/xblock-group-project.git@0.1.1#egg=xblock-group-project==0.1.1
 -e git+https://github.com/edx-solutions/xblock-adventure.git@7bdeb62b1055377dc04a7b503f7eea8264f5847b#egg=xblock-adventure
--e git+https://github.com/open-craft/xblock-poll.git@1.5.1#egg=xblock-poll==1.5.1
+-e git+https://github.com/open-craft/xblock-poll.git@1.6.1#egg=xblock-poll==1.6.1
 -e git+https://github.com/edx/edx-notifications.git@0.6.8#egg=edx-notifications==0.6.8
--e git+https://github.com/open-craft/problem-builder.git@2.11.1#egg=xblock-problem-builder==2.11.1
+-e git+https://github.com/open-craft/problem-builder.git@2.11.2#egg=xblock-problem-builder==2.11.2
```

Comment:
@zamanafzal  CircleCI test are failing. It seems like the problem-builder release "2.11.2" does not exist. The actual release name is "v2.11.2". Change this and then wait for CircleCI tests.